src/moneroInfo.js

```diff
@@ -2,14 +2,11 @@
 
 import { type EdgeCurrencyInfo } from 'edge-core-js/types'
 
-import type { MoneroSettings } from './moneroTypes.js'
+import type { MoneroUserSettings } from './moneroTypes.js'
 
-const otherSettings: MoneroSettings = {
-  mymoneroApiServers: ['https://edge.mymonero.com:8443']
-}
-
-const defaultSettings: any = {
-  otherSettings
+const defaultSettings: MoneroUserSettings = {
+  enableCustomServers: false,
+  moneroLwsServer: 'https://edge.mymonero.com:8443'
```

Comment:
`moneroLwServer` maybe? I get LW being an abbreviation for lightwallet, but I don't get the "s".